app/models/group.rb

```diff
@@ -10,18 +10,17 @@
 class Group < ApplicationRecord
   belongs_to :leader, class_name: 'User'
   belongs_to :draw
-  has_many :memberships
+  has_many :memberships, dependent: :delete_all
```

Comment:
Should we test this somehow? It's pretty lucky that I caught it by just playing with the app manually... a good reminder that automated tests don't always cover every aspect of the app. Where would this test go? On the model? e.g.

```ruby
it 'destroys dependent memberships` do
  group = FactoryGirl.create(:special_group)
  memberships = group.memberships
  group.destroy
  expect(Membership.find(memberships.map(&:id))).to be_empty
end
```

I guess that could work, let me know what you think and I'll add it (and we can do something similar for users).